src/util/language_file.h

```diff
@@ -75,10 +85,23 @@ class language_filest:public messaget
 
   bool interfaces(symbol_tablet &symbol_table);
 
+  bool has_lazy_method(const irep_idt &id)
+  {
+    return lazymethodmap.count(id);
+  }
+
+  void convert_lazy_method(
+    const irep_idt &id,
+    symbol_tablet &symtab)
+  {
+    return lazymethodmap.at(id)->convert_lazy_method(id, symtab);
```

Comment:
Yes, it is always an error to try to do second-stage conversion on a method that hasn't undergone first stage. Added a comment.